# 1. 目标数据
float32 x_goal
float32 y_goal
---
# 2. 反馈数据
float32 response
---
# 3. 连续反馈
float32 progress_bar